src/main/java/edu/harvard/iq/dataverse/authorization/AuthenticationServiceBean.java

```diff
@@ -380,14 +380,7 @@ public ApiToken findApiTokenByUser(AuthenticatedUser au) {
         }
         if (tokens.size() == 1) {
             // Normal case - one token that may or may not have expired
-            ApiToken token = tokens.get(0);
-            if (token.getExpireTime().before(latest)) {
-                // Don't return an expired token which is unusable, delete it instead
-                em.remove(token);
-                return null;
-            } else {
-                return tokens.get(0);
-            }
+            return tokens.get(0);
```

Comment:
so the idea is this will just return the expired token and it weon't work but user will then no to reset it?

Reply:
Yes - the UI already has code to warn the user that it has expired.